test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10138,6 +10139,30 @@ def forward(self, t: Tensor) -> Tuple[Tensor, Tensor]:
         x = torch.zeros(1)
         self.run_test(torch.jit.script(M()), (x,))
 
+    # This test is for shapeValueMap.
+    def test_shape_value_map(self):
```

Comment:
Double-checking: did you confirm this test fails without your fixes?

Reply:
I mentioned this in the previous comments:

https://github.com/pytorch/pytorch/pull/68203#discussion_r748478493
The assertion does not fail before and after the change for this particular unit test. Both works, so I don't add in test_pytorch_onnx_onnxruntime.py but just in test_operators.py.
This unit test is extracted from a large model. The original large model fails because the shape does not propagate fully. 

I don't find an example and it is not easy to have an unit test (from a large model) so that the test fails before and works with this PR.

Reply:
If this test didn't fail before, then it doesn't make sense to add it as part of this PR.

Can we write a test that directly asserts on the shapes in the TorchScript IR nodes?
Something that asserts that the statement in the comment on line 1023 of test/onnx/test_operators.py is True?

Reply:
Sure, changed to `assert loaded_model.graph.output[0].type.tensor_type.shape.dim[1].dim_value == 128`